sdks/python/scripts/run_pytest.sh

```diff
@@ -35,6 +35,11 @@ if [[ $pytest_args =~ "-m" ]] || [[ $posargs =~ "-m" ]]; then
   exit 1
 fi
 
+# strip leading/trailing quotes from posargs because it can get double quoted as its passed through.
```

Comment:
Just curious, is the quotes the actual cause prevented `--ignore` to work https://github.com/apache/beam/pull/24204#discussion_r1058662550 ?

Reply:
Yeah - it tried to parse `'--ignore=apache_beam/coders` as an argument instead of the `ignore` flag.